test-plone-4.3.x.cfg

```diff
@@ -9,5 +9,5 @@ versions = versions
 
 
 [versions]
-setuptools = 28.2.0
-zc.buildout = 2.11.0
+setuptools = 41.6.0
```

Comment:
Please move that to a separate pull request where you can explain in deep why this is necessary 😉